components/tools/OmeroPy/src/omero/testlib/__init__.py

```diff
@@ -93,6 +93,16 @@ class ITest(object):
     # If the new user created in setup_class should own their group
     # Can be overriden by test instances
     DEFAULT_GROUP_OWNER = False
+    # If the new user created in setup_class should
+    # be member of system group. This is to be
+    # set to True if the new user should be Restricted Admin
+    # Can be overriden by test instances
+    DEFAULT_SYSTEM = False
+    # If the new user created in setup_class is system group member
+    # default value None makes the new user to Full Admin
+    # For Restricted Admin with no privileges, set the value to []
+    # Can be overriden by test instances
+    DEFAULT_PRIVILEGES = None
```

Comment:
You can blame me for that one, I know little more Python than @pwalczysko. :smiley:

Reply:
@joshmoore : Not sure I understand now, in your second comment here you suggested to use round brackets instead of square brackets. Which I implemented. Your commit now seem to revert on this chenge ? Or is there some granularity (doc vs non-doc) which I am missing ?

Reply:
Apologies, I was trying to correct another line while preparing for a call and inadvertently changed this one. Give me just a bit to try to correct my mistake.

Reply:
Force pushed the correction. Please let me know what you think.